src/ui/config.ts

```diff
@@ -0,0 +1,2 @@
+export const DROPDOWN_MAX_SEARCH_RESULTS = 10
+export const DROPDOWN_TAGS_DISPLAY_LIMIT = 2
```

Comment:
These constants are not environment variables, but are declared at build time.
We want them to be replaced at runtime.

Reply:
Yes. As mentionned by @fredericvilcot, consider these values as environment variables injected as runtime. See `APP_ENV` declared in [src/env.d.ts](https://github.com/okp4/dataverse-portal/blob/732902ca92edb2a1437fcc60fd3ea90aae972248/src/env.d.ts).